capstone/src/main/webapp/script.js

```diff
@@ -18,7 +18,10 @@
  * restaurant data
  */
 function getRestaurants() {
-  fetch('/restaurants').then(function(response) {
+  const params = new URLSearchParams();
+  params.append('query', '');
+
+  fetch('/search', {method: 'GET', body: params}).then(function(response) {
```

Comment:
Is there a reason we're specifying the search parameters in the body instead of as [query parameters](https://en.wikipedia.org/wiki/Query_string)? 

For example, google supports query strings:
https://www.google.com/search?q=kittens

Reply:
Oh, I think I just did that incorrectly. I'll fix that.

Reply:
I ended up removing the body altogether, as it wasn't needed.